wuvt/static/css/wuvt.css

```diff
@@ -218,8 +224,15 @@ header #mainheader #robot #stream_btn {
     margin-left: 39px;
 }
 
-header #mainheader #robot #volume_btn {
+header #mainheader #robot #vol_btn_wrapper {
+    width: 43px;
+    height: auto;
```

Comment:
The reason for the wrapper div is it's the only way to disable a button while also having a disabled icon

Reply:
I see... It wasn't clear to me that the button was disabled. I think it's a bit confusing to have a regular volume icon that is established to be volume control, but simply not have it be clickable. Cursors aren't even visible on mobile so there really is no way to know why the button doesn't work. From an accessibility standpoint, disabling buttons in pure CSS has a lot of issues. Using the screen reader on my phone, the button is still treated a regular button and I have no indication I'm not supposed to use it. Funny enough, that's fine because since you're only blocking _pointer_ events so I can still activate it with the screen reader (I can also activate it with keyboard navigation on my PC, again when pointer events are disabled).

Personally, this seems like a job for JS. And since the buttons are only there if JS is already present and the scripts have successfully loaded, none of the issues of JS failing to load/not supported/intentionally disabled are relevant. I would add a listener to the window resize event and below your threshold change the icon as well as set the HTML disabled attribute on the button. Side note: normally I prefer aria-disabled since disabled completely takes them out of the page and they can't even be inspected by keyboard or screen reader users, but in this case that seems the correct way to treat it. As if it was never a button at all.